examples/gno.land/r/demo/msha1/msha1_test.gno

```diff
@@ -0,0 +1,18 @@
+package msha1
```

Comment:
You can move this test to the stdlibs folder, or you can keep it here and create an official example folder to test the stdlibs. If you choose to keep it here, I suggest moving it outside of the `r/` folder. What about `gno.land/p/demo/stdlibs_tests`?

Reply:
Moving test to stdlibs folder looks more straightforward to me. I'll move it